src/rvsdg/egglog_optimizer/subst.egg

```diff
@@ -89,4 +90,4 @@
   )
   ( 
     (union (SubstVecVecOperandHelper (VVO vec) x v i) (VVO vec))
-  ) :ruleset subst)
+  ) :ruleset subst)
```

Comment:
What's the diff on this line? (GitHub issue not code review issue)

Reply:
it's just that there's no longer a newline at the end of the file
(we should probably add the newline back so that we don't have unnecessary diffs like this)

Reply:
fixed!